RefRed/configuration/export_xml_config.py

```diff
@@ -144,8 +144,9 @@ def main_part(self):
 
             str_array.append('   <auto_q_binning>False</auto_q_binning>\n')
 
-            angleValue = str(self.parent.ui.angleOffsetValue.text())
-            angleError = str(self.parent.ui.angleOffsetError.text())
+            # The angle offset is currently not exposed in the UI, so we set it to 0.
+            angleValue = 0
+            angleError = 0
```

Comment:
I got an error when trying to save a configuration:
```
Traceback (most recent call last):
  File "/home/u5z/projects/RefRed/RefRed/main.py", line 507, in save_as_configuration
    o_save_config.run()
  File "/home/u5z/projects/RefRed/RefRed/configuration/saving_configuration.py", line 43, in run
    ExportXMLConfig(parent=self.parent).save(self.filename)
  File "/home/u5z/projects/RefRed/RefRed/configuration/export_xml_config.py", line 196, in save
    self.main_part()
  File "/home/u5z/projects/RefRed/RefRed/configuration/export_xml_config.py", line 150, in main_part
    str_array.append('   <angle_offset>' + angleValue + '</angle_offset>\n')
TypeError: can only concatenate str (not "int") to str
Aborted
```

Reply:
Sorry. Should be fixed now.